src/CoreBundle/Resources/views/FlashMessage/render.html.twig

```diff
@@ -8,18 +8,57 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
+
 {% for type in sonata_flashmessages_types() %}
     {% set domain = domain is defined ? domain : null %}
-    {% for message in sonata_flashmessages_get(type, domain) %}
+    {% set messages = sonata_flashmessages_get(type, domain) %}
+    {% if messages|length > 1 %}
+        <div class="alert alert-{{ type|sonata_status_class }} alert-dismissible collapsed-box">
+            <button
+                    type="button"
+                    class="close"
+                    data-dismiss="alert"
+                    aria-hidden="true"
+                    aria-label="{{ 'message_close'|trans({}, 'SonataCoreBundle') }}">
+                &times;
+            </button>
+            <input type="checkbox" class="read-more-state" id="toggle-more" />
+            <script>$(document).ready(()=> {$('#toggle-more').iCheck('destroy')})</script>
```

Comment:
I know this isn't nice but i need to disable iCheck for this case. shall i put this in the `base.js` file?

Reply:
If it is needed on every page, yes, otherwise, just create a new JS file?

Reply:
it is needed on every page but only for `#toggle-more` element